TACC-Cloud/geoapi
# task/WG-95: update asset backup

## Overview: ##

Update assets backup to backup NFS assets.  

## Related Jira tickets: ##

* [WG-195](https://tacc-main.atlassian.net/browse/WG-195)

## Summary of Changes: ##

## Testing Steps: ##
1. check running job https://jenkins01.tacc.utexas.edu/job/Geoapi%20(assets%20backup)/
2. check configure on that Jenkins workflow
3.  I checked the size and glanced at the file listing of the already finished staging part on ranch.  Looks to be good but you could optionally also look at that.
